src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -523,6 +523,8 @@ export default {
       }
     } finally {
       this.isLoading = false;
+      await this.fetchRecommendInfo();
```

Comment:
These two lines can be outside the finally block, and in its own try catch block?

Reply:
`fetchRecommendInfo` has its own try-catch in mixins

Reply:
Setting of `isRecommendationLoading` should be put inside that function then?

Reply:
Sure